src/app/components/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './Button.module.css';
+
+interface ButtonProps {
+  style: 'warning' | 'success' | 'primary' | 'secondary' | 'accent';
+  text: string;
+  icon?: JSX.Element;
+  onClick?: () => void;
+}
+
+export default function Button({
+  style,
+  text,
+  icon,
+  onClick,
+}: ButtonProps): JSX.Element {
+  return (
+    <button
+      onClick={onClick}
+      className={`${styles.button} ${style === 'accent' && styles.accent} ${
+        style === 'primary' && styles.primary
+      } ${style === 'secondary' && styles.secondary} ${
+        style === 'warning' && styles.warning
+      } ${style === 'success' && styles.success}`}
```

Comment:
😵this makes me dizzy, please access styles with `styles[style]` ;)
https://javascript.info/object#computed-properties

Reply:
Nice. I dont know that. Thank you =)